dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/SessionFactoryFactory.java

```diff
@@ -16,10 +16,7 @@
 import org.slf4j.LoggerFactory;
 
 import javax.sql.DataSource;
-import java.util.List;
-import java.util.Map;
-import java.util.SortedSet;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
Please don't use wildcard imports.

See https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports if you're using IntelliJ IDEA.